index.html

```diff
@@ -38,5 +37,131 @@ <h1>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project2.png" alt="">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project3.png" alt="">
+            <h2 class="project-title">Tonic</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
+
+        <article class="project">
+            <img src="./img/project4.png" alt="">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="stats">
+                <span class="company">CANOPY</span>
+                <span class="role-and-year">Back End Dev</span>
+                <span class="role-and-year">2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+
+            <ul class="languages">
+                <li class="language">html</li>
+                <li class="language">css</li>
+                <li class="language">javascript</li>
+            </ul>
+
+            <button class="btn">See Project</button>
+        </article>
```

Comment:
- [ ] The list of projects should use a ul tag to group them, and li per project

Reply:
It's nice to see that you worked on the requested change. good job!